subscriptions.php

```diff
@@ -0,0 +1,75 @@
+<?php 
+require_once("template.php");
+require_once("config.php");
+require_once("classes/constellation.php");
+Template::render_header("Subscriptions");
+$tg_user = getTelegramUserData();
+
+if($tg_user !== false){
+
+	if(isset($_GET['add'])){
+		$service = $_GET['add'];
+		$query = $mysqli->query("SELECT * FROM subscribers WHERE telegramID=" . $tg_user['id']);
+		while($subscriber = $query->fetch_assoc()){
+		  $subscriberID = $subscriber['subscriberID'];
+		}
+		$stmt = $mysqli->prepare("INSERT INTO services_subscriber VALUES (NULL,?, ?)"); 
+        $stmt->bind_param("ii", $subscriberID, $service);
+        $stmt->execute();
+		$query = $stmt->get_result();
+		header("Location: index.php?do=subscriptions");
+	}
+
+	if(isset($_GET['remove'])){
+		$service = $_GET['remove'];
+		$query = $mysqli->query("SELECT * FROM subscribers WHERE telegramID=" . $tg_user['id']);
+		while($subscriber = $query->fetch_assoc()){
+		  $subscriberID = $subscriber['subscriberID'];
+		}
+		$stmt = $mysqli->prepare("DELETE FROM services_subscriber WHERE subscriberIDFK = ? AND serviceIDFK = ?");
+		$stmt->bind_param("ii", $subscriberID, $service);
+		$stmt->execute();
+		$query = $stmt->get_result();
+		header("Location: index.php?do=subscriptions");
+	}
+
+	$query = $mysqli->query("SELECT services.id, services.name, subscribers.subscriberID, subscribers.telegramID
```

Comment:
Maybe use `FROM services as se LEFT join services_subscribers as se_su` ... `LEFT JOIN subscribers as su` or something like that? This looks way too long to me :slightly_smiling_face:

Reply:
Will check tomorrow. I'm not really a pro at mysql

Reply:
Let me know if you need any help, it's no problem for me to change it :wink:

Reply:
Do you mean:
```sql
SELECT id, name, subscriberID, telegramID FROM services as se LEFT JOIN services_subscriber as se_su ON se_su.serviceIDFK = se.id LEFT JOIN subscribers as su ON se_su.subscriberIDFK = su.subscriberID WHERE su.telegramID =
```
?

It's a little bit shorter...

Reply:
Yeah like this, it's just my personal preference to use shorter names when I need to reference tables explicitly. Looking at this again though I don't think that is necessary.

Reply:
ah okey. Since I've already added it to the code, I will keep it ;)